test/pointing_party/vote_calculator_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule PointingParty.VoteCalculatorTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+
+  describe "calculate_votes/1" do
+    setup do
+      points_map = fixed_map(%{points: integer(1..5)})
+
+      metas_map =
+        fixed_map(%{
+          metas: list_of(points_map, length: 1)
+        })
+
+      user = nonempty(map_of(string(:alphanumeric), metas_map))
+      [user: user]
```

Comment:
I didn't catch this the first time, but `users` would be a better name for this data. It's a map of one or more users.